netstats/conn.go

```diff
@@ -9,24 +9,53 @@ import (
 	"github.com/segmentio/stats"
 )
 
-func NewConn(c net.Conn, client stats.Client, tags ...stats.Tag) net.Conn {
+type ConnTag int
+
+const (
+	TagProtocol   ConnTag = 1 << 0
+	TagLocalAddr  ConnTag = 1 << 1
+	TagLocalPort  ConnTag = 1 << 2
+	TagRemoteAddr ConnTag = 1 << 3
+	TagRemotePort ConnTag = 1 << 4
```

Comment:
We probably will, but I'm willing to wait for them to complain before making the change :P